src/draw/nema_gfx/lv_draw_nema_gfx_config.h

```diff
@@ -0,0 +1,96 @@
+/******************************************************************************
```

Comment:
Just like the provided HAL, I think this also should be optional, as it is specific to ST HW and it is part of the TouchGFX distribution.

Reply:
Do we have any updates on this PR ? Can we proceed with merging so I can create another one on top of that?

Reply:
@kisvegabor, @liamHowatt  Do we have any updates on this PR ? Can we proceed with merging so I can create another one on top of that?

Reply:
Sorry for the delay. The config has been removed and I confirmed it compiles and runs.

You can see this line in the HAL that depended on the config which I had to remove: https://github.com/lvgl/lvgl/pull/7174/commits/bfe661fe1a4756159b369c07840fef6de904eb2c#diff-c1eb38101bd561ccd3f945b89474ead4114c86fd20d25ab4df85802332941509L68

It's only relevant to compilers that define `__ICCARM__`, seemingly the IAR compiler. I can't claim whether this will have a negative affect on those users or not.